build-system/tasks/performance/README.md

```diff
@@ -2,6 +2,6 @@
 
 Use `gulp performance` to run this test. It measures performance for the current branch compared to the current release. By default, the command first needs to compile minified runtime and components from the current branch by executing the `gulp dist` task. To skip this step, use the `--nobuild` flag, but the task will throw an error if the dist files are missing. Only the runtime and components used by documents from the configured URLs are compiled.
 
-The `config.json` file contains settings. If you set `headless` to `false`, you will be able to watch the test run in Chrome. The default value for `runs` is 100, which takes ~10 minutes to run, and generates results with ~5% margin of error. Differences larger than this should be investigated. The `urls` array of strings must be set before running the test, and each URL will be tested separately.
+The `config.json` file contains settings. If you set `headless` to `false`, you will be able to watch the test run in Chrome. The default value for `runs` is 100, which takes ~10 minutes to run, and generates results with ~5% margin of error. Differences larger than this should be investigated. The `urls` array of strings must be set before running the test, and each URL will be tested separately. If you wish to test `urls` hosted locally, use `gulp --cdn` to serve the binaries from cdn.
```

Comment:
I don't think we should ask all to use `gulp serve --compiled` when testing locally.  
It should be up to user to decide to use `gulp serve --compiled` or `gulp serve --cdn`.

Reply:
Well if you use the `--compiled` version the task won't work for locally served pages. It only works with `--cdn`.

Reply:
Why doesn't it worked with `--compiled`?

Reply:
To download the amp scripts ([local](https://github.com/ampproject/amphtml/blob/master/build-system/tasks/performance/rewrite-script-tags.js#L40) and [master](https://github.com/ampproject/amphtml/blob/master/build-system/tasks/performance/rewrite-script-tags.js#L60)), we look for the CDN_URL (`https://cdn.ampproject.org/`), and that's only present when serving with `--cdn`

Reply:
Hmm I see, that's interesting.

Reply:
I see. I wonder why don't we use the local server proxy `/proxy`.